schemas/nl.basjes/yauaa_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,191 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a context generated by the YAUAA enrichment after parsing the user agent",
+    "self": {
+        "vendor": "nl.basjes",
+        "name": "yauaa_context",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "DeviceClass": {
```

Comment:
I think we generally use `camelCase`

Reply:
I let the field names generated by YAUAA API but I can replace the first letter by a small one